webrtc-stats.html

```diff
@@ -2005,6 +2006,17 @@ <h2>
                   Since it is only given for active tracks, the incremental exposure is small.
                 </p>
               </dd>
+              <dt>
+                <dfn>active</dfn> of type <span class=
+                "idlMemberType">boolean</span>
+              </dt>
+              <dd>
+                <p>
+                  Indicates whether this <a>RTP stream</a> is configured to be sent or disabled.
+                  Note that an active stream can still not be sending, e.g. when being limited by
```

Comment:
do you think we will need a counter for the number of times the status flips between `active` and `limited`.

Reply:
This flag isn't supposed to flip on its own and only modifiable via setParameters which we don't count.
For limited see #650 but that is partially captured by qualityLimitationDurations.

Reply:
Having a counter for number of times setParameters flips this value makes little sense and is not very useful in my opinion. This metric is useful on its own and in fact very much needed for making sense of simulcast stats. Today you can't really tell "what is the highest resolution layer currently being sent?" or similar questions. We desperately need it for making sense of simulcast :)

Reply:
Agree that qualityLimitationDurations is enough, this is more about the effects of the current limitation